lua/wikis/worldoftanks/FilterButtons/Config.lua

```diff
@@ -0,0 +1,42 @@
+---
+-- @Liquipedia
+-- wiki=worldoftanks
+-- page=Module:FilterButtons/Config
+--
+-- Please see https://github.com/Liquipedia/Lua-Modules to contribute
+--
+
+local Game = require('Module:Game')
+local Tier = require('Module:Tier/Utils')
+
+local Config = {}
+
+---@type FilterButtonCategory[]
+Config.categories = {
+	{
+		name = 'liquipediatier',
+		property = 'liquipediaTier',
+		load = function(category)
+			category.items = {}
+			for _, tier in Tier.iterate('tiers') do
+				table.insert(category.items, tier.value)
+			end
+		end,
+		defaultItems = {'1', '2', '3'},
+		transform = function(tier)
+			return Tier.toName(tier)
+		end,
+		expandKey = "game",
+	},
+	{
+		name = 'game',
+		property = 'game',
+		expandable = true,
+		items = {'worldoftanks', 'mir tankov'},
```

Comment:
Use the same approach as AoE instead (#5627)

Reply:
Applied the AoE version